txcache/virtualAccountRecord.go

```diff
@@ -7,21 +7,19 @@ import (
 )
 
 type virtualAccountRecord struct {
-	initialNonce    core.OptionalUint64
-	initialBalance  *big.Int
-	consumedBalance *big.Int
+	initialNonce   core.OptionalUint64
+	virtualBalance *virtualAccountBalance
```

Comment:
Keep a separate note to check, during final benchmarking, whether it would be optimal (speed) to have this as a "value", not as a "pointer" (since it's small, it holds two mere pointers within).